Networking/Networking/Mapper/ProductVariationsBulkUpdateMapper.swift

```diff
@@ -34,21 +38,17 @@ struct ProductVariationsBulkUpdateMapper: Mapper {
 /// This entity allows us to do parse all the things with JSONDecoder.
 ///
 private struct ProductVariationsEnvelope: Decodable {
```

Comment:
nit: similar to above, `ProductVariationsContainerEnvelope` would be more clear

Reply:
Also done in 35dc049a6ac7f9ca4584a99c1e59cef688afd671.